operator/apis/machinelearning.seldon.io/v1/seldondeployment_webhook_test.go

```diff
@@ -394,6 +391,41 @@ func TestMetricsPortAddedTwoComponentSpecsTwoContainers(t *testing.T) {
 	g.Expect(volFound).To(BeTrue())
 }
 
+func TestOverrideMetricsPortName(t *testing.T) {
+	os.Setenv(ENV_PREDICTIVE_UNIT_METRICS_PORT_NAME, "myMetricsPort")
```

Comment:
We actually set them up as global variables so we can inject them in the test, you can do it similar to how we do it in the other tests

Reply:
I didn't find a test overriding env vars, can you point me to one?